x/ibc/07-tendermint/keeper/verify.go

```diff
@@ -14,52 +14,11 @@ import (
 	ibctypes "github.com/cosmos/cosmos-sdk/x/ibc/types"
 )
 
-var _ clientexported.ClientState = ClientState{}
-
-// ClientState from Tendermint tracks the current validator set, latest height,
-// and a possible frozen height.
-type ClientState struct {
-	// Client ID
-	ID string `json:"id" yaml:"id"`
-	// Latest block height
-	LatestHeight uint64 `json:"latest_height" yaml:"latest_height"`
-	// Block height when the client was frozen due to a misbehaviour
-	FrozenHeight uint64 `json:"frozen_height" yaml:"frozen_height"`
-}
-
-// NewClientState creates a new ClientState instance
-func NewClientState(id string, latestHeight uint64) ClientState {
-	return ClientState{
-		ID:           id,
-		LatestHeight: latestHeight,
-		FrozenHeight: 0,
-	}
-}
-
-// GetID returns the tendermint client state identifier.
-func (cs ClientState) GetID() string {
-	return cs.ID
-}
-
-// ClientType is tendermint.
-func (cs ClientState) ClientType() clientexported.ClientType {
-	return clientexported.Tendermint
-}
-
-// GetLatestHeight returns latest block height.
-func (cs ClientState) GetLatestHeight() uint64 {
-	return cs.LatestHeight
-}
-
-// IsFrozen returns true if the frozen height has been set.
-func (cs ClientState) IsFrozen() bool {
-	return cs.FrozenHeight != 0
-}
-
 // VerifyClientConsensusState verifies a proof of the consensus state of the
 // Tendermint client stored on the target machine.
-func (cs ClientState) VerifyClientConsensusState(
+func (k Keeper) VerifyClientConsensusState(
```

Comment:
Why are we moving these functions to the keeper?

Reply:
we are going to use a prefix store for each client